book/security.md

```diff
@@ -704,12 +700,29 @@ if you forget a form---and relying on every website out there to do it
 right is a pipe dream. It'd be better for the browser to provide a
 fail-safe backup.
 
+::: {.further}
+One unusual attack, similar in spirit to cross-site request forgery,
+is [click-jacking][clickjacking]. This attack involves including an
+external site in a transparent `iframe` positioned over a
+seemingly-innocent site. The user thinks they are clicking around the
+innocent site, but they actually take actions on the external site.
+Nowadays, sites can prevent this with the [`frame-ancestors`
+directive][csp-frame-ancestors] to `Content-Security-Policy` or the
+older [`X-Frame-Options` header][x-frame-options].
```

Comment:
There is also an occlusion detection API as part of IntersectionObserver, targeted at this use case
specifically:

https://web.dev/intersectionobserver-v2/

Reply:
I guess this is like a more programmable `X-Frame-Options`, if you do in fact need to allow iframe embedded on unknown sites? Do you think we should write about this?